### ISSUE TITLE
Loading an example erases a user's work

### ISSUE THREAD

**Issue by gormster** [owenthereal/jqplay#120]
Given that examples can be loaded via query string, it seems kind of silly that the examples overwrite the existing work that a user has done, without so much as pushing a history state. I mean sure, it's unlikely a user has spent hours crafting a perfect query in this little playground, but it's annoying to lose even a few minutes work, especially if I can't instantly recall what it was I had written.

The examples should work if I command-click (or control-click) to open them in a new tab, and they shouldn't erase work I've already done with no way to get it back. I can think of no simpler way to do this than by simply making each example link contain its example data in the query string.

**Comment by owenthereal** [comment]
I agree that having a way to track the history of query modifications—similar to how Google Docs handles version history—would significantly enhance the user experience. The ability to revert changes easily could make the platform much more intuitive and user-friendly.
